src/Bindings/ManualBindings_World.cpp

```diff
@@ -110,6 +110,73 @@ static int tolua_cWorld_ChunkStay(lua_State * tolua_S)
 
 
 
+static int tolua_cWorld_ForEachLoadedChunk(lua_State * tolua_S)
+{
+	// Exported manually, because i don't know what i'm doing...
```

Comment:
It has to be exported manually because tolua doesn't support converting functions to functor types.